src/NServiceBus.WindowsPerformanceCounters/Performance/ReceiveStatisticsConfig.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace NServiceBus
+{
+    using WindowsPerformanceCounters;
+
+    /// <summary>
+    /// Provide configuration options for statistics performance counters.
+    /// </summary>
+    public static class ReceiveStatisticsConfig
+    {
+        /// <summary>
+        /// Enables the NServiceBus statistics performance counters.
+        /// </summary>
+        /// <param name="endpointConfiguration">The <see cref="EndpointConfiguration" /> instance to apply the settings to.</param>
+        public static void EnablePerformanceCounters(this EndpointConfiguration endpointConfiguration)
+        {
+            Guard.AgainstNull(nameof(endpointConfiguration), endpointConfiguration);
+            endpointConfiguration.EnableFeature<ReceiveStatisticsFeature>();
```

Comment:
Something will probably need to be renamed. Our `ReceiveStatisticsFeature` has the same name and namespace as the one in core.

For message property encryption, we put everything in a new namespace: `NServiceBus.Encryption.MessageProperty`. Should we do the same here?